src/allotropy/parsers/utils/strict_xml_element.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from xml.etree import ElementTree
+
+# xml fromstring is vulnerable so defusedxml version is used instead
+from defusedxml.ElementTree import fromstring  # type: ignore[import-untyped]
```

Comment:
Add "types-defusedxml >= 0.7.0.20240218" to requirements and you can lose these untyped comments!